src/algebra/category/Module/kernels.lean

```diff
@@ -63,4 +63,63 @@ lemma has_kernels_Module : has_kernels (Module R) :=
 lemma has_cokernels_Module : has_cokernels (Module R) :=
 ⟨λ X Y f, has_colimit.mk ⟨_, cokernel_is_colimit f⟩⟩
 
+open_locale Module
+
+local attribute [instance] has_kernels_Module
+local attribute [instance] has_cokernels_Module
+
+/--
+The categorical kernel of a morphism in `Module`
+agrees with the usual module-theoretical kernel.
+-/
+noncomputable def kernel_iso_ker {G H : Module.{v} R} (f : G ⟶ H) :
+  kernel f ≅ Module.of R (f.ker) :=
+limit.iso_limit_cone ⟨_, kernel_is_limit f⟩
+
+-- We now show this isomorphism commutes with the inclusion of the kernel into the source,
+-- stating this both at the morphism level, and elementwise
+
+@[simp] lemma kernel_iso_ker_inv_kernel_ι {G H : Module.{v} R} (f : G ⟶ H) :
+  (kernel_iso_ker f).inv ≫ kernel.ι f = f.ker.subtype :=
+by { convert (limit.iso_limit_cone_inv_π _ _); refl, }
```

Comment:
```suggestion
limit.iso_limit_cone_inv_π _ _
```

Reply:
Huh! Presumably I started with some dumb version of the statement that didn't quite match, and the `convert` was needed, then fixed it and didn't notice the proof simplified...